cmd/cc-log-parser/README.md

```diff
@@ -0,0 +1,47 @@
+# `cc-log-parser`
+
+`cc-log-parser` is a tool that combines log files from various Clear
+Containers components. It sorts the logfiles by timestamp and re-displays
+them, adding a time delta showing how much time has elapsed between each log
+entry.
+
+## Component logfiles
+
+The primary two logs that are passed to the tool are the [runtime global
+log](https://github.com/clearcontainers/runtime#debugging) and the [proxy
+log](https://github.com/clearcontainers/proxy#debugging).
```

Comment:
**note:**
This wrapping doesn't break anything because it is within the rendered text, however, note that you can ignore 78char rules when it is a link taking you over the max. It's fine as is but should the 78 char mark hit outside of the square brackets, you would wrap text before or after the full
\[runtime global log](https://github.com/clearcontainers/runtime#debugging)